app/api/entities/specs/entitySchema.spec.js

```diff
@@ -191,13 +202,22 @@ describe('entity schema', () => {
           entity.metadata.name = [{ value: 10 }];
           await expectError(customErrorMessages[propertyTypes.text], ".metadata['name']");
         });
+
+        it('should fail if value is a string that exceeds the lucene term byte-length limit', async () => {
+          entity.metadata.name = [{ value: largeField }];
+          await expectError('maximum field length exceeded', ".metadata['name']");
+        });
       });
 
       describe('markdown property', () => {
         it('should fail if value is not a string', async () => {
           entity.metadata.markdown = [{ value: {} }];
           await expectError(customErrorMessages[propertyTypes.markdown], ".metadata['markdown']");
         });
+        it('should fail if value is a string that exceeds the lucene term byte-length limit', async () => {
+          entity.metadata.markdown = [{ value: largeField }];
+          await expectError('maximum field length exceeded', ".metadata['markdown']");
+        });
```

Comment:
While this, indeed, apparently checks for the text and markdown property, this tests will all pass if the "IF" testing the property type is removed.  This is a case of implementing something that is not really tested beforehand.

Where is the test that defines that that particular "IF" for the property type is required?  It doesn't exist.  So, please, define if it is needed and either test is or remove it.

Thanks.

Reply:
you're right @RafaPolit, this condition is not necessary by tests neither the running app